configs/KAKUTEF7MINIV3/config.h

```diff
@@ -80,30 +81,27 @@
 #define CAMERA_CONTROL_PIN   PB3
 
 #define TIMER_PIN_MAPPING \
-    TIMER_PIN_MAP( 0, PB4 , 1,  0) \
-    TIMER_PIN_MAP( 1, PB5 , 1,  0) \
-    TIMER_PIN_MAP( 2, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 3, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 4, PB3 , 1,  0) \
-    TIMER_PIN_MAP( 5, PB6 , 1,  0) \
-    TIMER_PIN_MAP( 6, PB7 , 1,  0) \
-    TIMER_PIN_MAP( 7, PA8 , 1,  2) \
-    TIMER_PIN_MAP( 8, PA3 , 3, -1)
+    TIMER_PIN_MAP( 0, MOTOR1_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 1, MOTOR2_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 2, MOTOR3_PIN        , 1,  0) \
+    TIMER_PIN_MAP( 3, MOTOR4_PIN        , 1,  0) \
+    TIMER_PIN_MAP( 4, MOTOR5_PIN        , 1,  0) \
+    TIMER_PIN_MAP( 5, MOTOR6_PIN        , 1,  0) \
+    TIMER_PIN_MAP( 6, CAMERA_CONTROL_PIN, 1,  0) \
```

Comment:
```suggestion
    TIMER_PIN_MAP( 6, CAMERA_CONTROL_PIN, 1,  -1) \
```

Reply:
Done for all KAKUTE targets, :1st_place_medal: